de/1/math_operations.md

```diff
@@ -1,6 +1,28 @@
 <!-- Add translation for the following page: https://vyper.fun/#/1/math_operations
 Do NOT change the code below. The below code runs the code editor -->
+# Chapter 3: Mathematische Operationen
 
+Mathematik in Vyper ist ziemlich unkompliziert. Die folgenden Operationen sind die gleichen wie in den meisten anderen Programmiersprachen:
+
+- Addition: `x + y`
+- Subtraktion: `x - y`
+- Multiplikation: `x * y`
+- Division: `x / y`
+- Divison mit Rest: `x % y` (Zum Beispiel `13 % 5` ist `3`, weil beim Dividieren von `13` durch `5`, bleibt der Rest `3`)
```

Comment:
Update from `Divison mit Rest` to `Modul / Rest`

Reply:
Well, I think the original term should be better. https://de.wikipedia.org/wiki/Division_mit_Rest

Reply:
In my oppinion, we should use the original term 

https://de.wikipedia.org/wiki/Division_mit_Rest